### DISCUSSION TITLE
Multiple mod loaders?

### DISCUSSION THREAD

**Discussion by SirenGlitch** [PrismLauncher/PrismLauncher#695] [Q&A] [answered]
Very quick and simple question, it appears that I can install 2 mod loaders (eg. Forge and Fabric) on one instance, does this work or does it replace the other one? I am new to MMC and Prism so sorry if I ask a lot of questions in future

**Comment by DioEgizio** [comment, marked as answer]
no, you can't run both forge and fabric/quilt at the some time, it's just not gonna work

**Reply by SirenGlitch** [reply]
then why does it not grey out the buttons to install them once you have forge installed?

**Reply by TheKodeToad** [reply]
Forge and LiteLoader works together

**Reply by benjamin051000** [reply]
I suppose in _theory_, it's possible to have multiple mod loaders present? So technically we shouldn't grey the buttons out. I guess it's currently up to you to install multiple and see if it works or not. 🤷🏼

**Comment by Ari-43** [comment]
It may be a good idea to have a warning of some sort so new, unsuspecting
users don't accidentally install multiple loaders without understanding
that there could be issues.
